testsuite/features/step_definitions/command_steps.rb

```diff
@@ -406,6 +406,9 @@
       log "#{time_spent / 60.to_i} minutes out of #{timeout / 60.to_i} waiting for '#{channel}' channel to be synchronized" if ((time_spent += checking_rate) % 60).zero?
       sleep checking_rate
     end
+  rescue Timeout::Error => e
+    log e.message
+    raise ScriptError, "Timeout occured while syncing channel: #{channel}"
```

Comment:
Why we need this begin statement catching `StandardError` and `Timeout` while in fact that's already addressed as part of `repeat_until_timeout` function?


See this:

https://github.com/NamelessOne91/uyuni/blob/master/testsuite/features/support/commonlib.rb#L121-L126

Reply:
Right, we can probably remove the error handling from the caller and just pass `$build_validation` as `dont_raise', if we intend to suppress errors

Reply:
Exactly.